app-backend/api/src/main/scala/user/Auth0User.scala

```diff
@@ -35,7 +37,7 @@ final case class Auth0User(
     created_at: Option[String],
     updated_at: Option[String],
     identities: Option[Json],
-    // app_metadata: Option[Json],
+    app_metadata: Option[Json],
```

Comment:
:thinking: why was this removed, and why do we need it again?

Reply:
Uhhh I don't know why it was removed. I brought it back for debug info, but it doesn't need to stay.

Reply:
Lol the commit with this was from _exactly three years ago_